packages/editor-ui/src/components/forms/ResourceSharingFilterSelect.ee.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<n8n-select
```

Comment:
this is not the same as n8n-user-select ?

Reply:
Migrated to n8n-user-select (added support for disabled entries as well).